lib/themes/dosomething/paraneue_dosomething/js/components/onboard.js

```diff
@@ -0,0 +1,21 @@
+const React = require('react');
+const ReactDOM = require('react-dom');
+
+import Carousel from './carousel.js';
+
+const Onboard = React.createClass({
+  render: function() {
+    return (
+      <div className="onboard">
+        <Carousel />
+      </div>
+    );
+  }
+});
+
+if (localStorage.getItem('dosomething_enable_onboarding') == "true") {
```

Comment:
Yeah, I'm working on adding a `dsUser` to the `Drupal.settings` object so we have access to a current user object via that settings object.

@deadlybutter What we talked about using local storage for was to store whether the member user already saw the onboarding experience or not. It would be a simple implementation and not needing a feature flag and database record for storing that a user has seen the item already for this beta. If it turns out we go a different direction with the beta for onboarding, then we'd have some Drupal cleanup and a random database record to deal with, whereas the cleanup with using local storage would be virtually nothing.

For _actually_ showing the onboarding or not across all campaigns and user... yeah, that should be a feature flag in the Drupes.